### PR TITLE
Adding monitor for disk status check

### PR DESCRIPTION
**Pull request by rohit-dimagi**
##### SUMMARY

* Adding monitor for disk status check

##### ENVIRONMENTS AFFECTED

* ICDS
##### ISSUE TYPE

- Change Pull Request

### CODE REVIEW

**Comment by rohit-dimagi** [comment]
It's easy to detect failure on db but would be useful for other server.s

File: src/commcare_cloud/manage_commcare_cloud/monitors/disk_rw_status.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+env_key: environment.name
+id: 11866550
+message: |-
+  {{#is_alert}}
+  {{device.name}} drive is mounted as read-only file system
+  on {{host.name}}
+  {{#is_match "environment.name" "icds"}} @slack-alerts-icds {{/is_match}}
+  << notification_block >>
```
Comment (snopoke) [inline src/commcare_cloud/manage_commcare_cloud/monitors/disk_rw_status.yml]: don't need this if it's only for ICDS

File: src/commcare_cloud/manage_commcare_cloud/monitors/disk_rw_status.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  silenced: {}
+  include_tags: true
+  thresholds:
+    warning: 1
+    ok: 1
+    critical: 1
+  new_host_delay: 300
+  notify_no_data: false
+  renotify_interval: 0
+query: '"disk.read_write".over("environment:icds").by("host").last(2).count_by_status()'
```
Comment (snopoke) [inline src/commcare_cloud/manage_commcare_cloud/monitors/disk_rw_status.yml]: should this also group by device?

File: src/commcare_cloud/manage_commcare_cloud/monitors/disk_rw_status.yml (hunk 1 above)
Comment (rohit-dimagi) [reply src/commcare_cloud/manage_commcare_cloud/monitors/disk_rw_status.yml]: Removed.

File: src/commcare_cloud/manage_commcare_cloud/monitors/disk_rw_status.yml (hunk 2 above)
Comment (rohit-dimagi) [reply src/commcare_cloud/manage_commcare_cloud/monitors/disk_rw_status.yml]: Added.

**Pull request merged**